src/main/java/com/minecolonies/coremod/entity/citizen/VisitorCitizen.java

```diff
@@ -743,7 +740,6 @@ protected void dropEquipment()
         }
     }
 
-    // TODO:REMOVE DEBUG
```

Comment:
while I think its fine to remove this, please remove the entire overriden function here, all of it is debug code :D